src/Surfnet/StepupGateway/GatewayBundle/Controller/GatewayController.php

```diff
@@ -44,7 +44,7 @@ public function ssoAction(Request $httpRequest)
         $redirectBinding = $this->get('surfnet_saml.http.redirect_binding');
 
         try {
-            $originalRequest = $redirectBinding->processUnsignedRequest($httpRequest);
+            $originalRequest = $redirectBinding->processSignedRequest($httpRequest);
```

Comment:
Hmm, I incorrectly assumed the change to unsigned requests was on purpose. :flushed: